app/assets/stylesheets/modules/_flow-grid.scss

```diff
@@ -0,0 +1,69 @@
+// These two constants must match those in flow_grid.js.coffee
+$grid-max: 30;
+$cell-max: 8;
+
+$cell-min: 4;
+$grid-min: $grid-max - $cell-max + 1;
+
+@mixin stellar-flow() {
+  @for $g from $grid-min through $grid-max {
+    .of-#{$g} {
+      padding-top: (100% / $g) * 6;
+    }
+    @for $c from $cell-min through $cell-max {
+      .c-#{$c}.of-#{$g} {
+        width: (100% / $g) * $c;
+      }
+    }
+  }
+}
+
+.flow-grid {
+  padding: 5px;
+
+  &:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+}
+
+@include stellar-flow();
+
+.grid-cell {
+  position: relative;
```

Comment:
Properties should be ordered background-color, border, box-sizing, float, overflow, position